op-e2e/actions/proofs/isthmus_fork_test.go

```diff
@@ -103,10 +103,10 @@ func testIsthmusActivationAtGenesis(gt *testing.T, testCfg *helpers.TestCfg[any]
 	require.Equal(t, got, reproduced.Hash())
 	require.True(t, ok, "CheckBlockHash must pass")
 
-	safeBlock := env.Sequencer.L2Safe()
+	safeBlock := env.BatchMineAndSync(t)
```

Comment:
wondering if we can remove the `env.Sequencer.ActL2PipelineFull(t)` call in line 78? no batching happens before, and no safe head is gotten afterwards.